runtime/keymap/arabic-qwerty.vim

```diff
@@ -0,0 +1,90 @@
+scriptencoding utf-8
+
+" Arabic phonetic keyboard from http://arabic.omaralzabir.com/
+" at https://github.com/suryaya/Arabic-Phonetic-Keyboard-Linux
+" See also https://r.nf/r/learn_arabic/comments/etj8x4/i_made_an_arabic_phonetic_keyboard_for_linux/
+
+
+" Use this short name in the status line.
+let b:keymap_name = "ara"
+
+loadkeymap
+
+1 ١ " Arabic_1
```

Comment:
The third field is a comment by definition so it doesn't need the `"` comment char.  Line comments do still need it though.

I know the keymap files aren't formatted consistently so it's really up to you.